Src/DeUrgenta.Certifications.Api/CommandHandlers/UpdateCertificationHandler.cs

```diff
@@ -29,19 +33,26 @@ public async Task<Result<CertificationModel>> Handle(UpdateCertification request
                 return Result.Failure<CertificationModel>("Validation failed");
             }
 
-            var certification = await _context.Certifications.FirstAsync(c => c.Id == request.CertificationId, cancellationToken);
+            var certification = await _context.Certifications
+                .FirstAsync(c => c.Id == request.CertificationId, cancellationToken);
             certification.Name = request.Certification.Name;
             certification.IssuingAuthority = request.Certification.IssuingAuthority;
             certification.ExpirationDate = request.Certification.ExpirationDate;
 
             await _context.SaveChangesAsync(cancellationToken);
-
+            
+            var photoUrl = await _storage.SaveAttachmentAsync(certification.Id, 
+                request.UserSub, 
```

Comment:
null pointer exception if user does not add photo :(

Reply:
fixed